src/harness/fourslash.ts

```diff
@@ -4359,4 +4395,11 @@ namespace FourSlashInterface {
     export interface CompletionsAtOptions {
         isNewIdentifierLocation?: boolean;
     }
+
+    export interface VerifyCodeFixOptions {
+        description: string;

```

Comment:
Should `description` and `newContent` be optional as well?

Reply:
Not if we want to enforce that these are tested.